pkg/k8/deployments/translate.go

```diff
@@ -165,8 +165,13 @@ func createSyncthingVolume(d *appsv1.Deployment, dev *model.Dev) {
 		d.Spec.Template.Spec.Volumes = []apiv1.Volume{}
 	}
 
-	syncVolume := apiv1.Volume{Name: dev.GetCNDSyncVolume()}
+	for _, v := range d.Spec.Template.Spec.Volumes {
+		if v.Name == dev.GetCNDSyncVolume() {
+			return
+		}
+	}
 
+	syncVolume := apiv1.Volume{Name: dev.GetCNDSyncVolume()}
```

Comment:
could this affect the scenario where we have two containers in the same deployment, and we turn both on?

Reply:
no, they use different volume names